.github/CONTRIBUTING.md

```diff
@@ -71,22 +72,44 @@ Documentation for running and updating our existing tests, as well as the code f
 
 Changes must be proposed and reviewed **individually** - long lists of tickets or changesets from individual WordPress versions don't give us what we need in order to plan and understand each change well.
 
+If you're not sure about any of that, then it's a good idea to ask first. A good way is to create an issue for the specific change you're interested in, along with links to the relevant WordPress changesets and tickets, and any other information you have about how the change works.
+
 There are some changes that we already know we want to backport because they fit into our plans for future versions of ClassicPress. You can find some examples under the [`WP backport` label](https://github.com/ClassicPress/ClassicPress/labels/WP%20backport).
 
-Otherwise, if you're interested in backporting a change from WordPress, then please explain in a pull request or issue what the specific change is, how it works, and why it's something that makes sense for ClassicPress to adopt. Often there is a lot of good discussion about each change in the relevant WordPress Trac tickets, and it makes the maintainers' job much easier to see this discussion summarized and linked.
+### Tips for a good Backport PR.
+- Give the PRs a clear title. Explain in a pull request or issue what the specific change is, how it works and why ClassicPress in particular should adopt it. Often there is a lot of good discussion about each change in the relevant WordPress Trac tickets. It makes the maintainers' job much easier to see this discussion summarized and linked.
+- Explain what testing has been done on the PR and what may be left to do. Screenshots, gif, video and text instructions documenting the tests can be very useful. Those tell other contributors looking at the PR how to verify the changes. "This was tested in WordPress" **is not enough** to meet our review criteria, see [Review Criteria](#review-criteria) above.
+- Look to make sure the backport is done correctly, with attribution to the original WP committer. 
+- If there are merge conflicts during the backport, check if they have been resolved correctly by comparing the final changes from the full PR against the corresponding WP changeset(s). Remember to fix these **in a separate commit**. If you cannot resolve the conflicts, ask for help in the [Core slack channel](https://www.classicpress.net/join-slack/) or just push your branch as-is and we'll take care of it!
 
-If you're not sure about any of that, then it's a good idea to ask first. A good way is to create an issue for the specific change you're interested in, along with links to the relevant WordPress changesets and tickets, and any other information you have about how the change works.
+### Making a Backport PR.
+When you're ready to backport a code change, make a new branch to your ClassicPress fork:
```

Comment:
> make a new branch to your ClassicPress fork:

This sounds like the next thing will show how to make a branch in the fork, but the next thing is about identifying a changeset (sort of).  Does the script make a branch? If so, say so. If not, put the command to make the branch.

Reply:
https://github.com/ClassicPress/ClassicPress/pull/710#discussion_r619825795